packages/expo-dev-launcher/ios/EXDevLauncherController.m

```diff
@@ -502,12 +521,22 @@ -(NSDictionary *)getBuildInfo
   NSString *sdkVersion = [self getUpdatesConfigForKey:@"EXUpdatesSDKVersion"];
   NSString *appVersion = [self getFormattedAppVersion];
   NSString *appName = [[NSBundle mainBundle] objectForInfoDictionaryKey: @"CFBundleDisplayName"] ?: [[NSBundle mainBundle] objectForInfoDictionaryKey: @"CFBundleExecutable"];
-
+  
+  // TODO - this could be moved to the expo-updates repo and installed as a dev extension
+  // url structore for EASUpdates: `http://u.expo.dev/{appId}`
+  // this url field is added to app.json.updates when running `eas update:configure`
+  // the `u.expo.dev` determines that it is the modern manifest protocol
+  NSString *updatesUrl = [self getUpdatesConfigForKey:@"EXUpdatesURL"];
+  NSURL *url = [NSURL URLWithString:updatesUrl];
+  NSString *appId = [[url pathComponents] lastObject];
```

Comment:
I think this has been updated a bit since you've reviewed - I just moved this logic into `expo-updates` as a dev extension - you can see the changes here: 5a3dec59b3916ab4695455abac901c88d061d82c